routes/web.php

```diff
@@ -14,17 +14,18 @@
 */
 
 // Route::get('hello','HelloController@index');
-Route::get('/', function () {
-    return view('welcome');
-});
-Route::get('hello',function() {
-    return '<html><body><h1>Hello</h1><p>This is sample page.
-    </p></body></html>';
-});
-Route::get('/top', function () {
-    return view('top');
-});
+// Route::get('/', function () {
+//     return view('welcome');
+// });
+// Route::get('hello',function() {
+//     return '<html><body><h1>Hello</h1><p>This is sample page.
+//     </p></body></html>';
+// });
+// Route::get('/top', function () {
+//     return view('top');
+// });
 
+Route::get('/top/{id}', 'UserController@top');
```

Comment:
UserControllerでも良いかなと思いましたが、URL部分もtopにしているので下記に修正しましょうか。
```suggestion
Route::get('/top/{id}', 'TopController@top');
```

Reply:
上記の変更に伴って、
`app/Http/Controllers/UserController.php` も `app/Http/Controllers/TopController.php`  にリネームしましょう。
リネームする際には、フォルダから直接変更するのではなく、 `git mv` コマンドを使ってリネームするようにしてください。